Workbooks/Azure Web Application Firewall/WAF monitoring/WAF Monitoring.workbook

```diff
@@ -105,8 +103,8 @@
             "type": 4,
             "isRequired": true,
             "value": {
-              "durationMs": 2764800000,
-              "endTime": "2022-12-30T21:32:00.000Z"
+              "durationMs": 5707080000,
+              "endTime": "2023-01-06T18:50:00.000Z"
             },
```

Comment:
this appears to be a custom time range?  like from today back a certain amount of time, a previous version was looking for specific times in december?  i presume you really want this to be something like "last 24 hours", not a specific start and end time?

Reply:
Are you referring to the default value for Time range ? if yes, then yes the last 24hours as the landing value makes sense.